### PR TITLE
Feature: add order to holding teams

### PR DESCRIPTION
**Pull request by ahmadsalimi**

### CODE REVIEW

File: backend/api/serializers.py [hunk 1]
```diff
@@ -96,7 +96,9 @@ class Meta:
 class HoldingTeamSerializer(ModelSerializer):
     class Meta:
         model = HoldingTeam
-        fields = '__all__'
+        fields = [
+            'id', 'name', 'wss', 'staff'
```
Comment (emranbm) [inline backend/api/serializers.py:100]: What's hiding here?

Comment (ahmadsalimi) [reply backend/api/serializers.py:100]: `order`. it's not necessary for the client, because the response is already sorted.

Comment (emranbm) [reply backend/api/serializers.py:100]: Hmm, but I think budanesh khali as lotf nist :)))

Comment (ahmadsalimi) [reply backend/api/serializers.py:100]: :)))))))))))))))
OK. I will add it again.

**Review by emranbm — APPROVED** [review]

**Pull request merged**